mqtt/mqtt-broker/src/auth/authorization.rs

```diff
@@ -193,14 +193,14 @@ mod tests {
 
     use matches::assert_matches;
 
-    use super::*;
+    use crate::auth::{Activity, Authorizer, DefaultAuthorizer, Operation};
 
     fn connect() -> mqtt3::proto::Connect {
-        proto::Connect {
+        mqtt3::proto::Connect {
```

Comment:
nit: `use mqtt3::proto;`
to be consistent with the rest of changes.

Reply:
fixed